Source/Border.js

```diff
@@ -50,7 +50,7 @@ export class Border extends WrappedObject {
   */
   set flatColor(value) {
     // TODO : Create internal overloader that routes color conversion based on type (hex, int etc)
-    var hexToNSColor = ColorHelper.colorFromHex(value);
+    var hexToNSColor = ColorHelper.hexToMSColor(value);
```

Comment:
This just may be me, and again should be defined by coding standards. But since it's early days and we don't have this I'll insert my opinion ;) I'm not a big fan of Hungarian notation and think that the end user of the API shouldn't have to know about the underlying objects like `MSColor`, so think this would be better as simply `hexToColor`.